src/swoop/api/db.py

```diff
@@ -0,0 +1,32 @@
+import asyncpg
+from .config import Settings
+
+async def connect_to_db(settings: Settings) -> None:
+    """Connect to Database."""
+    db = DB()
+    readpool = await db.create_pool(settings.reader_connection_string, settings)
+    writepool = await db.create_pool(settings.writer_connection_string, settings)
+    return readpool, writepool
+
+async def close_db_connection(pools) -> None:
+    """Close connection."""
+    await pools.readpool.close()
+    await pools.writepool.close()
+
+class DB:
+    """DB class that can be used with context manager."""
+
+    connection_string = None
+    _pool = None
+    _connection = None
+
+    async def create_pool(self, connection_string: str, settings):
```

Comment:
I don't see self referenced here. What is the point of this `DB` class versus just making `create_pool` a standalone function?